pandas/tests/dtypes/test_missing.py

```diff
@@ -444,7 +442,7 @@ def test_array_equivalent(dtype_equal):
     )
     # The rest are not dtype_equal
```

Comment:
Nit: Could update this to say the second case is equal?

Reply:
i think this is in reference to the dtype_equal keyword